src/SUMMARY.md

```diff
@@ -103,6 +103,7 @@
     - [Release Process](./release/process.md)
     - [Rollup Procedure](./release/rollups.md)
     - [Triage Procedure](./release/triage-procedure.md)
+    - [Issue Triaging](./release/issue-triaging.md)
```

Comment:
Does this really belong in the “Release” section? As a newcomer, that's not where I'd expect to find information about how the issue tracker is managed.